tests/Data/ProgrammesDb/EntityRepository/CoreEntityRepository/FindDescendantsByTypeTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository::FindDescendantsByType
+ * @group test
+ */
+class FindDescendantsByTypeTest extends AbstractDatabaseTest
+{
+    public function testProgrammesChildrenGetCorrectClips()
+    {
+        $this->loadFixtures(['MongrelsFixture']);
+        $dbAncestryId = $this->getAncestryFromPersistentIdentifier('b010t19z', 'Brand');
+
+        $clipsUnderProgramme = $this->getEntityManager()
+            ->getRepository('ProgrammesPagesService:CoreEntity')
+            ->FindDescendantsByType($dbAncestryId, 'Clip', 100, 0);
+
+        $expectedClipsPids = ['p00h64pq', 'p00hv9yz', 'p008k0l5', 'p008k0jy', 'p008nhl4'];
+        $this->assertEquals($expectedClipsPids, array_column($clipsUnderProgramme, 'pid'));
+        // all clips are streamable
+        $this->assertEquals(
```

Comment:
`$this->assertNotContains(false, array_column($clipsUnderProgramme, 'streamable'));` might be a bit shorter and more readable